contracts/pool/src/methods/withdraw.rs

```diff
@@ -39,7 +40,7 @@ pub fn withdraw(
     let s_token = STokenClient::new(env, &reserve.s_token_address);
     let debt_token = DebtTokenClient::new(env, &reserve.debt_token_address);
```

Comment:
Looks like debt token client used here only for getting it's address. Can we get address from reserve and avoid client instantiating?

Reply:
Nice catch! Thanks!